runtime/lib/backend/lightning/lightning_dynamic/LightningSimulator.cpp

```diff
@@ -374,9 +374,9 @@ void LightningSimulator::Counts(DataView<double, 1> &eigvals, DataView<int64_t,
     // into a bitstring.
     for (size_t shot = 0; shot < shots; shot++) {
         std::bitset<CHAR_BIT * sizeof(double)> basisState;
-        size_t idx = 0;
+        size_t idx = numQubits - 1;
         for (size_t wire = 0; wire < numQubits; wire++) {
-            basisState[idx++] = li_samples[shot * numQubits + wire];
+            basisState[idx--] = li_samples[shot * numQubits + wire];
```

Comment:
Would you also fix this for LightningKokkos?

Reply:
Yes done